src/widgets/widget.ts

```diff
@@ -34,6 +35,10 @@ export class Widget {
 
 
     // Search single element
+    public bySystelabTestId(dataTestId: string): ElementFinder {
+        return this.elem.byCSS(`[${Test.SYSTELAB_TEST_ID_ATTRIBUTE}="${dataTestId}"]`);
```

Comment:
What about calling to the new method added at ElementFinder level > bySystelabTestId()

Reply:
Applied